common.js

```diff
@@ -9,7 +9,7 @@
 const endpoint = process.env.TEST_ENDPOINT || 'http://localhost:1026';
 
 const accEndpoint = process.env.ACC_ENDPOINT || 'http://localhost:3000';
-const notifyEndpoint = process.env.NOTIFY_ENDPOINT || `${accEndpoint}/acc`;
+const notifyEndpoint = process.env.NOTIFY_ENDPOINT || 'http://host.docker.internal:3000/acc';
```

Comment:
So, you've decided to hardcode the receiver of notifications?
Might be a good idea.
Make sure you change this everywhere.
Also, the env var for the receiver I think is part of the README. If so, it should be removed from there.

Reply:
I checked.
README.md says this:

```
export NOTIFY_ENDPOINT=<The notification endpoint>. Example: `http://host.docker.internal:3000/acc`
```

Reply:
I have changed it in README and I hope every thing is good now.